code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -372,7 +372,7 @@
 	if(air_master.current_cycle%3==1 && length(touchable_items))
 
 		//Burn all the mobs or add them to the exclusion list
-		for(var/mob/living/carbon/human/T in (touchable_items))
+		for(var/mob/living/T in (touchable_items))
 			if((T.status_flags & GODMODE) || !T.digestable)
```

Comment:
It's good that this changed, since it should be `/mob/living`, but this does mean that those will still stay forever.

The only code that actually deletes mobs from bellies still checks to see if it's a human, on 387 (the check) and 430 (the qdel).

Also while you're there change 391's "T.stat & DEAD" because stat is not a bitfield, it should be ==

Reply:
Changing that check you mention on 387 works and gives everything the desired behavior: living things get digested once they die and indigestible things get set aside.

PR amended.